src/face_tracking/README.md

```
@@ -19,7 +19,7 @@ python sanity_check.py
 
 
 
-To run the face tracking, run the following command in the CLI in the face_tracking directory
+To run full face tracking with the drone, use the [`follow_face.py`](../example_exercises/follow_face.py) exercise script. As noted in the [example exercises guide](../example_exercises/README.md), it must be run from the `src` directory:
 ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)